doc/release-notes.md

```diff
@@ -1,3 +1,22 @@
+# Schema Release `v0.6.0a9`
+
+Changes to `ga4gh/schemas` `master` branch since version `0.6.0a8` (Oct 26, 2016)
+
+* Fixed bugs:
+   * Fixed typo in get: `/v0.6.0a8/variantannotationset/{variant_annotation_set_id}`
+   * Fix to be able to handle VCFs with genotype == `./.`
+* Upgrade to use protobuf release 3.1
+* Introduced a pip installable schemas package called `ga4gh-schemas`. We have also 
+   created pip installable packages for a support library called `ga4gh-common`
+   and a client library module called `ga4gh-client`.
+* Introduced a schemas package release to Maven Central, which can be found by 
+   searching for ga4gh on the Maven site.
+* Changed the name of the biosample terms to track be consistent with the use
```

Comment:
Also, I headed over to https://search.maven.org/#search%7Cga%7C1%7Cga4gh and see a release from fnothaft and opencb, but not GA4GH.